src/shared/migrations/041-add-more-slot-groups-attributes.ts

```diff
@@ -0,0 +1,33 @@
+import { DataTypes } from 'sequelize';
+
+/**
+ * Adds the requiredDLCs column to the MissionSlots table
+ */
+module.exports = {
+    up: async (queryInterface: any): Promise<void> => {
+        await queryInterface.addColumn('missionSlotGroups', 'radioFrequency', {
+            type: DataTypes.DOUBLE,
+            allowNull: true
```

Comment:
Please add `defaultValue` attributes to all migrations as well (where appropriate). For most `allowNull: true` columns, `defaultValue: null` is fitting - updating existing records can fail otherwise since the migration does not know what to insert.

(commenting this once here - relevant to multiple places in the migrations)